kernelci/dashboard
# fix: alter log-downloader error status code to 400

Close #559

## How to test
- Run the backend (after exporting the necessary environment variables) with `poetry run python3 manage.py runserver`
- Send and GET request to `api/log-downloader` and check if the status code is 400 with `curl 'localhost:8000/api/log-downloader/?log_download_url=example.com' -o - -I`
